sbpy/photometry/core.py

```diff
@@ -777,7 +876,8 @@ class LinearPhaseFunc(DiskIntegratedModelClass):
 
     _unit = 'mag'
     H = Parameter(description='Absolute magnitude')
-    S = Parameter(description='Linear slope (mag/rad)')
+    S = Parameter(description='Linear slope (mag/deg)')
+    input_units = {'x': u.deg}
 
     @staticmethod
     def evaluate(a, H, S):
```

Comment:
can you add a short docstring here? Any function or method that does not have a leading underscore will show up in the API reference, so people might wonder what this function actually does. Even if it is a 'private' function/method, adding a docstring is nice for future debugging...

Reply:
Well, I don't fully understand it yet, but it appears to control the default unit of input `x`, based on the docs in `astropy.modeling.Model` and other examples that I could find online.  Will add this to the docstring.

Reply:
Well, how do we add docstring to attribute like this?  @mommermi   I tried below code but the docstring doesn't show up in `help(LinearPhaseFunc)`.

```
input_units = {'x': u.deg}
"""Default unit for independent variable"""
```

Reply:
I think via: `input_units.__doc__ = "documentation"`

Reply:
Nevermind: `AttributeError: 'dict' object attribute '__doc__' is read-only`

Reply:
Instead of docstring, I added comment lines in the class definition of `DiskIntegratedPhaseFunc`.